ArmyOfOnes/ArmyOfOnes/ViewControllers/CurrenciesListViewController.swift

```diff
@@ -0,0 +1,216 @@
+//
+//  CurrenciesListViewController.swift
+//  ArmyOfOnes
+//
+//  Created by Fabian Mariño on 11/8/16.
+//  Copyright © 2016 Fabian Mariño. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+enum LayoutConstraints: CGFloat {
+    case inputOffset = 14,
+    topInset = 43,
+    tableTopInset = 70
+}
+
+enum textFieldLimit: Int {
+    case max = 15
+}
+
+class CurrenciesListViewController: UITableViewController {
+    
+    var didSetupConstraints = false
+    var dollarQuantity = 1.0
+    
+    let quantityInput: UITextField = {
```

Comment:
When you need to initialize the variable in line with its declaration it's better to use [lazy instantiation](https://www.hackingwithswift.com/example-code/language/what-are-lazy-variables). It allows you to, later on, update this variable if needed.